### PR TITLE
Initialize cURL only once and support cookies

### PR DESCRIPTION
**Pull request by oxan**
This initializes cURL only once and enables it cookiefile options, supporting cookies to be used.

### CODE REVIEW

File: lib/Tivoka/Client/Connection/Http.php [hunk 1]
```diff
@@ -61,6 +63,25 @@ public function __construct($target) {
...
+                'Content-Type: application/json',
+                'Connection: Close'
+            );
+            foreach($this->headers as $label => $value) {
+                $headers[] = $label . ": " . $value;
+            }
+
+            $this->ch = curl_init($this->target);
+            curl_setopt($this->ch, CURLOPT_RETURNTRANSFER, true);
+            curl_setopt($this->ch, CURLOPT_HTTPHEADER, $headers);
```
Comment (marcelklehr) [inline lib/Tivoka/Client/Connection/Http.php]: Shouldn't this be moved to send(), because the user may want to add headers *after* constructing?

**Comment by marcelklehr** [comment]
Looks good, though!

**Comment by oxan** [comment]
Yeah, you're right. I've pushed a fix for that.

**Comment by marcelklehr** [comment]
Thank you! I'll add a descructor for good measure to avoid leaking curl handles.

**Pull request merged**